python/clx/workflow/splunk_alert_workflow.py

```diff
@@ -123,7 +123,8 @@ def __pivot_table(self, gdf, index_col, piv_col, v_col):
         index_list = gdf[index_col].unique()
         piv_gdf = cudf.DataFrame({index_col: list(range(len(index_list)))})
         piv_gdf[index_col] = index_list
-        for group in gdf[piv_col].unique():
+        piv_groups = gdf[piv_col].unique().to_pandas()
```

Comment:
Is there a way we can avoid using `to_pandas` here?

Reply:
The code before update was already transferring to host by the for loop. `cudf` no longer allows to the series object to be iterated implicitly which is why we are now calling `to_pandas` before iterating. Also, this is a temporary implementation of `pivot_table` until it's provided by cudf. Here is the feature request:

https://github.com/rapidsai/cudf/issues/1214

Reply:
@efajardo-nv Would love to hear your input on https://github.com/rapidsai/cudf/issues/1214#issuecomment-667160536 re: `pivot_table`. Thanks!